advanced-usage/index.md

```diff
@@ -0,0 +1,286 @@
+- Generic Named Value indicator
+    - clicking on this allows selecting which `NAMED_VALUE_FLOAT/INT` MAVLink message to track, out of
+      any that have been received so far (including custom ones)
+    - it is also possible to specify a display unit, a value multiplier, and an icon
+{{ easy_image(src="named-value-mini-widget-config", width=300, center=true) }}
+- Video recorder
+    - allows recording one of the available WebRTC streams, or the full Cockpit tab
+    - recording occurs in the browser of the display device (not onboard the vehicle)
+    - recordings are saved using the [mission name](#mission-name) and the starting timestamp
+- Joystick connection status indicator
+- Flight mode selector
+- GPS status indicator
+- [View](#views) selector
+
+## Configuration
+
+Cockpit's behaviour can be configured via the [burger menu](#burger-menu), with the following tabs:
+
+### General
+
+Connection configuration allows specifying custom endpoint addresses for Cockpit to communicate with.
+When Cockpit is hosted by a vehicle running BlueOS these are usually correct by default, but if using
+it standalone or connecting to some external services it may be necessary to specify different
+addresses, and refresh the page to establish the desired connection.
+
+{{ easy_image(src="../getting-started/general-config", width=600, center=true) }}
+
+### Joysticks
+
+Cockpit is intended to work with arbitrary joystick types, and allows mapping joystick buttons and axes
+to [`BTNn_FUNCTION`](https://docs.bluerobotics.com/ardusub-zola/software/autopilot/ArduSub-4.1/developers/parameters/#btnn-function-function-for-button)
+values for use in regularly sent [`MANUAL_CONTROL`](https://mavlink.io/en/messages/common.html#MANUAL_CONTROL)
+MAVLink messages, as well as to Cockpit Actions which can send commands to the vehicle or trigger interface
+events like like view switching or starting a video recording.
+
+{{ easy_image(src="../getting-started/joystick-config", width=600, center=true) }}
+
+Support is built in for simultaneous input from multiple sources, including multiple joysticks, and by
+default each joystick can provide up to 8 axis ranges and 32 buttons. As some caveats,
+
+- ArduSub only supports 16 independent buttons and 4 axis ranges in its `MANUAL_CONTROL` handling
+- When mapping the Z motion axis range, note that ArduSub uses 0 to +1000 for full reverse to full forwards,
+whereas other vehicle types use -1000 to +1000
+- Cockpit can assign a button to the "Shift" `BTNn_FUNCTION`, but is not currently capable of mapping
+a secondary "shifted" autopilot action to its other buttons
+    - This currently needs to be set up separately, either directly using vehicle parameters, or using
+    an alternative control station software like QGroundControl
+- Cockpit is currently only capable of mapping to `BTNn_FUNCTION` values that have already been assigned to
+a button, which may need to be done in advance using parameters or QGroundControl
```

Comment:
Worth mentioning that editing those parameters is doable on BlueOS, and also that we will allow this on Cockpit itself in about 1 week.

Reply:
I think that's ok as is, we can update the docs when the feature is available.